manifests/init.pp

```diff
@@ -89,6 +89,9 @@
 #                                           file('another_module/autosign.sh') or
 #                                           template('another_module/autosign.sh.erb')
 #
+# $autosign_source::                        If set, use this as the source for the autosign file,
+#                                           instead of autosign_content.
```

Comment:
If you specify both content and source, puppet will just reject it.

For example, `puppet apply -e 'file { "/tmp/foo": content => "foo bar\n", source => "/tmp/motd" }'` produces:

`Error: Validation of File[/tmp/foo] failed: You cannot specify more than one of content, source, target at line 1`

While there is certainly value in having a slightly more specific error message, this is something meant for people running the module outside of the Foreman Installer.  I'm not sure it's worth this much hand-holding.  People running the module stand alone should understand basic puppet resources.

This suggestion just seems like over kill to me.